lsdcGui.py

```diff
@@ -38,7 +38,7 @@
 import numpy as np
 import _thread #TODO python document suggests using threading! make this chance once stable
 import lsdcOlog
-from threads import VideoThread
```

Comment:
This change is related to the hutch cam thread may not be related to this pull request

Reply:
fixed in commit https://github.com/NSLS-II/lsdc/pull/247/commits/742f46502f2f357c70b5637f1fdc5368398b5968